pulsar/tests/test_e2e.py

```diff
@@ -0,0 +1,24 @@
+# (C) Datadog, Inc. 2022-present
+# All rights reserved
+# Licensed under a 3-clause BSD style license (see LICENSE)
+import pytest
+
+from datadog_checks.base.constants import ServiceCheck
+
+from .common import EXPECTED_METRICS, OPTIONAL_METRICS
+
+pytestmark = [pytest.mark.e2e]
+
+
+def test_check(dd_agent_check, instance):
+    aggregator = dd_agent_check(instance, rate=True)
+
+    aggregator.assert_service_check('pulsar.openmetrics.health', ServiceCheck.OK)
+
+    for metric in EXPECTED_METRICS:
+        aggregator.assert_metric(metric)
+
+    for metric in OPTIONAL_METRICS:
+        aggregator.assert_metric(metric, count=0)
```

Comment:
```suggestion
        aggregator.assert_metric(metric, at_least=0)
```

Reply:
This is a small nit but the tests were more "informative" before because they were asserting that the metrics were never collected, this is not really asserting anything now